### PR TITLE
Apply default Studio code style

### PR DESCRIPTION
**Pull request by liujoshua**
Applied Android Studio's default code format to make it easier to work with ResearchStack and  BridgeAndroidSDK as submodules.

See [MA-50](https://sagebionetworks.jira.com/browse/MA-50)

### CODE REVIEW

File: MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerResourceManager.java [hunk 1]
```diff
@@ -16,75 +16,65 @@
...
-    private static final String BASE_PATH_HTML        = "html";
-    private static final String BASE_PATH_JSON        = "json";
-    private static final String BASE_PATH_JSON_SURVEY = "json/survey";
-    private static final String BASE_PATH_PDF         = "pdf";
-    private static final String BASE_PATH_VIDEO       = "mp4";
-
-    public static final int PEM    = 4;
+    public static final int PEM = 4;
     public static final int SURVEY = 5;
+    private static final String BASE_PATH_HTML = "html";
```
Comment (EricSiegNW) [inline MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerResourceManager.java]: not sure I like these spacing changes, seems to make it less readable

File: MPower/gradle.properties [hunk 2]
```diff
@@ -1,21 +1,16 @@
 # Project-wide Gradle settings.
-
```
Comment (EricSiegNW) [inline MPower/gradle.properties]: really dislike these changes.. they are separate thoughts. Maybe add a # separator?

File: MPower/app/src/main/assets/html/learn_treatment.html [hunk 3]
```diff
@@ -1,39 +1,60 @@
...
     <div class="content">
 
-        <p><img class="displayed" src="file:///android_res/drawable/logo_disease_information_top.png" width="75" height="72" alt="App Logo" style="background-color: #FFFFFF; display:block;" /></p>
+        <p><img class="displayed"
+                src="file:///android_res/drawable/logo_disease_information_top.png" width="75"
+                height="72" alt="App Logo" style="background-color: #FFFFFF; display:block;"/></p>
         <p>&nbsp;</p>
-        <p>A physician will diagnose Parkinson disease from the medical history and a neurological examination. There is no lab test that will clearly identify the disease, but brain scans are sometimes used to rule out disorders that could give rise to similar symptoms.</p>
-        <p>People may be given levodopa and resulting relief of motor impairment tends to confirm diagnosis. The progress of the illness over time may reveal it is not Parkinson disease, and some authorities recommend that the diagnosis be periodically reviewed.
+        <p>A physician will diagnose Parkinson disease from the medical history and a neurological
```
Comment (EricSiegNW) [inline MPower/app/src/main/assets/html/learn_treatment.html]: line length stuff def makes it easier to read on github :)

File: MPower/app/src/main/assets/json/consent_section.json [hunk 4]
```diff
@@ -59,135 +59,134 @@
...
-  "sections":
-  [
+  "sections": [
     {
-      "sectionType" : "overview",
-      "sectionTitle" : "Welcome",
-      "sectionSummary" : "This overview explains the research study. After you learn about the study, you can choose if you would like to participate. This may take about 20 minutes to complete.",
-      "sectionLearnMoreButtonTitle" : "Learn more",
-      "sectionHtmlContent" : "consent_1welcome"
+      "sectionType": "overview",
```
Comment (EricSiegNW) [inline MPower/app/src/main/assets/json/consent_section.json]: currently these files are synced with iOS, should we reflect these changes back into iOS?

**Review by EricSiegNW — APPROVED** [review]
I'm fine with merging, but see the comments

File: MPower/app/src/main/assets/json/consent_section.json (hunk 4 above)
Comment (dephillipsmichael) [inline MPower/app/src/main/assets/json/consent_section.json]: consent_section is going to be gone with other PR, so I wouldn't worry to much for now.

File: MPower/gradle.properties (hunk 2 above)
Comment (liujoshua) [inline MPower/gradle.properties]: Agreed. Thanks for looking

File: MPower/app/src/main/assets/json/consent_section.json (hunk 4 above)
Comment (liujoshua) [inline MPower/app/src/main/assets/json/consent_section.json]: It'd be ideal if the json and html were synced, maybe versioned in a separate repo and included via git submodule or git subtree

**Comment by liujoshua** [comment]
Apparently I don't have write access to this repo, someone else can merge :)

**Pull request merged**